internal/serve/httphandler/login_handler_test.go

```diff
@@ -24,6 +26,23 @@ import (
 	"github.com/stellar/stellar-disbursement-platform-backend/stellar-auth/pkg/auth"
 )
 
+func authenticateSetup(
+	t *testing.T,
+	authenticatorMock *auth.AuthenticatorMock,
+	jwtManagerMock *auth.JWTManagerMock,
+	user *auth.User, password, userToken string,
+) {
+	t.Helper()
+	authenticatorMock.
+		On("ValidateCredentials", mock.Anything, user.Email, password).
+		Return(user, nil).
+		Once()
+	jwtManagerMock.
+		On("GenerateToken", mock.Anything, user, mock.AnythingOfType("time.Time")).
+		Return(userToken, nil).
+		Once()
+}
+
```

Comment:
Can you be more specific on this method name? Maybe `prepareMocksForLogin` would be more accurate?